AOJ/03/03_A.py

```diff
@@ -0,0 +1,2 @@
+for value in range(1000):
+ print ('Hello World')
```

Comment:
インデントが1になっていますので、他のコードと同様にインデントを2に変更してください。
```suggestion
  print ('Hello World')
```